src/@next/Banner/BannerStyle.ts

```diff
@@ -72,6 +72,27 @@ export const StyledBanner = styled.div`
     fill: ${Red.B93};
   }
 
+  &[data-type='fixed'],
+  &[data-type='fixed-noicon'] {
+    position: fixed;
```

Comment:
so in Figma it was written:
> Position - Fixed Below Header

was about to set `top: m px;` where m is the height of the header, but I realized it made the component unadaptive to different codebases if we do that...

Reply:
I think we should have a default gap from header by default, but I suggest to give a class name for the section which user can override easier later

Reply:
So, do you mean having some default value (let's say top: 50px), and just let the user pass in extra style prop (e.g., style={{ top: "20px" }}) in case they want to override the default one? something like this is okay? (if yes, then I'll confirm with Maeve the height of the header)

Reply:
We don't need to expose the style, what I meant is user can override them if they want with styled-component from their side, as long as the element is given a proper selector

Reply:
I don't think we need to expose the style, but just put some className to the part which users can override later